docs/modules/sql/pages/mapping-to-maps.adoc

```diff
@@ -120,7 +119,7 @@ For more information about this serialization option, see xref:serialization:com
 
 === JSON Objects
 
-If values are in the JSON format, configure the `valueFormat` field as the xref:working-with-json.adoc[`JSON` or `JSON-FLAT` type].
+If values are in the JSON format, configure the `valueFormat` field as the xref:working-with-json.adoc[`json` or `json-flat` type].
```

Comment:
Elsewhere we put types in uppercase. Why have we changed it to lowercase for JSON types?